pennylane_lightning_kokkos/src/simulator/MeasuresFunctors.hpp

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+#include <Kokkos_StdAlgorithms.hpp>
+
+#include "Util.hpp"
+
+namespace {
+using namespace Pennylane::Util;
+namespace KE = Kokkos::Experimental;
+} // namespace
+
+namespace Pennylane {
+namespace Functors {
+
+template <class Precision> struct getProbFunctor {
```

Comment:
We should add docstrings for all of these functors

Reply:
done